src/SW_Control.c

```diff
@@ -330,11 +334,34 @@ void SW_CTL_setup_domain(unsigned long userSUID,
         return; // Exit function prematurely due to error
     }
 
+    #if defined(SWNETCDF)
+    SW_NC_read(SW_Domain, LogInfo);
+    if(LogInfo->stopRun) {
+        return; // Exit function prematurely due to error
+    }
+    #endif
+
     SW_DOM_read(SW_Domain, LogInfo);
     if(LogInfo->stopRun) {
         return; // Exit function prematurely due to error
     }
 
+    #if defined(SWNETCDF)
+    if(FileExists(SW_Domain->PathInfo.InFilesNC[DOMAIN_NC])) {
+
+    } else {
+        SW_NC_create_domain_template(SW_Domain, LogInfo);
+        if(LogInfo->stopRun) {
+            return; // Exit prematurely due to error
+        }
+
+        LogError(LogInfo, LOGERROR, "Domain netCDF template has been created. "

```

Comment:
Let user know the file path to the domain template in addition to stating that "Domain netCDF template has been created. "